sunkit_pyvista/plotter.py

```diff
@@ -140,3 +144,38 @@ def plot_solar_axis(self, length=2.5, arrow_kwargs={}, **kwargs):
                          scale='auto',
                          **defaults)
         self.plotter.add_mesh(arrow, **kwargs)
+
+    def plot_field_lines(self, gong_map, lon, lat, radius, nrho=25, rss=2.5, **kwargs):
+        """
+        Plots the field lines from `pfsspy`.
+
+        Parameters
+        ----------
+        gong_map : `pfsspy.map.GongSynopticMap`
+            to produce the field lines
+        lon : `astropy.units.quantity`
+        lat : `astropy.units.quantity`
+        radius : `float`
+        nrho : `int`
+        rss : `int`
+        **kwargs :
+            Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
+        """
+        tracer = tracing.PythonTracer()
+        input_ = pfsspy.Input(gong_map, nrho, rss)
+        output_ = pfsspy.pfss(input_)
+
+        seeds = SkyCoord(lon, lat, radius*R_sun,
+                         frame=gong_map.coordinate_frame)
+
+        field_lines = tracer.trace(seeds, output_)
```

Comment:
These lines should all go in the example script, and the `plot_field_lines` method should just take `field_lines` and `**kwargs` as input. That way, the user has access to the PFSS solution if they want to trace different lines through it or do other things that aren't plotting the lines.

Reply:
yeah this makes more sense, I'll update the PR